src/components/views/rooms/MessageComposer.js

```diff
@@ -274,6 +316,15 @@ export default class MessageComposer extends React.Component {
                 </div>
             );
 
+            const locationButton = (
+                // Add option for either current or specifed location.
+                // TODO: Add icon
```

Comment:
Hm, in practice we probably are going to want a generic + button or something which provides a popup for the more exotic stuff like push-to-talk, locations, etc.  But we can handle that!